[NOT FOR MERGE]canaries/pcengines/canary-009-2023.txt: add Canary 009 (#72)
3mdeb/3mdeb-secpack: canaries/pcengines/canary-009-2023.txt

                ---===[ PC Engines Canary #9 ]===---

Statements
-----------

The 3mdeb firmware developers who have digitally signed this file [1]
state the following:

1. The date of issue of this canary is {issue_date}.

2. The 3mdeb Master Signing Key fingerprint is:

    1B57 85C2 965D 84CF 85D1  652B 4AFD 81D9 7BD3 7C54

3. The 3mdeb Open Source Firmware Master Key fingerprint is:

    EA61 CBAD DA1C 094A 2540  E596 028A 7527 64CB 97EC

> 4. PC Engines Open Source Firmware Release 4.19 Signing Key is signed

Reviewer: ```suggestion
4. PC Engines open-source firmware release 4.19 signing key is signed
```

by 3mdeb Open Source Firmware Master Key, which is signed by 3mdeb
Master Key.

5. No warrants have ever been served to us with regard to the PC Engines
firmware (e.g. to hand out the private signing keys or to introduce
backdoors).

6. Due to new coreboot 4.19 release the PC Engines firmware binaries are

> going to be signed with PC Engines Open Source Firmware Release 4.19

Reviewer: ```suggestion
going to be signed with PC Engines open-source firmware release 4.19
```

Signing Key with following fingerprint:

    05CF 36F1 66C3 D676 A08A  B70F 30A5 3DE2 F5A6 D89A

> 7. The PC Engines firmware version v4.19.0.1 is signed with PC Engines Open
> Source Firmware Release 4.19 Signing Key, which public signature should be

Reviewer: ```suggestion
7. The PC Engines firmware version v4.19.0.1 is signed with PC Engines 
open-source firmware release 4.19 signing key, which public signature should be
```

available on 3mdeb-secpack repository.

8. PC Engines firmware version v4.19.0.1 will be the last PC Engines firmware

> release made by 3mdeb on behalf of PC Engines.

Reviewer: It would be great to have, for further information please check docs.dasharo.com/....

Author: Any ETA for such information? Or should we just blindly add docs.dasharo.com and work on the info separately later?

Reviewer: ETA is release date. I think we can imagine the name of the md file in docs and structure where it would be and predict how link would look like. `https://docs.dasharo.com/variants/pcengines_apu2/announcement/` ?

Author: Yes, that's what I meant by `blidnly`

Author: Added: https://github.com/3mdeb/3mdeb-secpack/pull/72/commits/9ab994536ef59d89ff58f1222b45fe51391bad12#diff-138201b9ffd22ad757480d3458521db7def8c3b5ac14d366ad3eaf1705dbdbecR39

9. PC Engines support for open-source firmware on apu1 and apu2 series

> platforms has ended in August 2022. The support is not expected to be resumed.

Reviewer: Any commercial inquiries regarding PC Engines firmware please direct to ...

Author: contact@dasharo.com I suppose?

Reviewer: sure

Author: Added https://github.com/3mdeb/3mdeb-secpack/pull/72/commits/9ab994536ef59d89ff58f1222b45fe51391bad12#diff-138201b9ffd22ad757480d3458521db7def8c3b5ac14d366ad3eaf1705dbdbecR44

10. The canary was planned to be published in the first two weeks of July 2020.
Due to the termination of open-source firmware support, the release of the
canary has been delayed.

> 11. There will be no more canaries published fo rPC Engines open-source firmware

Reviewer: ```suggestion
11. There will be no more canaries published for PC Engines open-source firmware
```
